db_autorestore.md

```diff
@@ -0,0 +1,89 @@
+# Automated database update and restore with AWS Lambda functions
+
+The road map to cloud adoption can be difficult to set and implement, but once it is completed it offers a lot of flexibility, a lot of space for continuous improvement and a lot of room for creativity to build new solutions.
+Having automated processes helps your company to focus on what is important for the business and lets the developers experiment more and efficienlty optimize their work. 
+One of the latest things I've been working on involves a task that needed to be continued after the client's solution was already moved to the cloud. What I liked about it was that it is not only taking the advantage of being in the cloud, it also involves serverless technology that I consider to be the next level of cloud solutions.
+
+## Scenario
+Our customer has 2 different AWS accounts:
+ * Acceptance
+ * Production
+
+and all the changes on the databases in Production account are needed in Acceptance account every 2 weeks. The databases used are AWS DocumentDB and AWS Aurora.
```

Comment:
I would change this to: "All changes made in the production account should be (every two weeks) synchronised to their acceptance account for regression testing in the acceptance account." 
Or something like that.